fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -636,6 +636,12 @@ public class Config extends ConfigBase {
             varType = VariableAnnotation.EXPERIMENTAL)
     public static boolean enable_single_replica_load = false;
 
+    @ConfField(mutable = true, masterOnly = true, description = {
+            "对于 DUPLICATE KEY 表启用 shuffle 的最小 tablet 数量",
```

Comment:
Could you explain more about when should user set this config?

Reply:
Enable shuffle can bring both positive effect and negative effect.
Shuffle must be enabled for `UNIQ` and `AGG` tables for data consistency reasons.
But for `DUP` tables, we can do some trade-offs.
If shuffle is disabled, the load will be faster but it will use more memory.
If shuffle is enabled, the load will be slower but it will use less memory.

When loading into table with more buckets, it's more likely to be memory constrained.
So we choose to enable shuffle only for tables with certain amount buckets.